R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2165,6 +2165,14 @@ test_that("SQL error message is returned from JVM", {
   expect_equal(grepl("blah", retError), TRUE)
 })
 
+test_that("Default warehouse dir should be set to tempdir", {
+  # nothing should be written outside tempdir() without explicit user permission
+  inital_working_directory_files <- list.files()
```

Comment:
I'm referring to other tests in this test file, test_sparkSQL, that is calling to API that might already initialize the warehouse dir.

`sparkR.session()` is called at the top. Does this `createOrReplaceTempView` cause the warehouse dir to be created?

https://github.com/shivaram/spark-1/blob/25834109588e8e545deafb1da162958766a057e2/R/pkg/inst/tests/testthat/test_sparkSQL.R#L570